parts/linux/cloud-init/artifacts/mig-partition.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Apply MIG configuration on Nvidia A100 GPU
+
+[Service]
+Restart=on-failure
+ExecStartPre=/usr/bin/nvidia-smi -mig 1
+ExecStart=/bin/bash /opt/azure/containers/mig-partition.sh {{GetGPUInstanceProfile}}
+
+[Install]
+WantedBy=multi-user.target
+#EOF
```

Comment:
I don't believe systemd artifact files should contain EOF here, have you tested this? 

I just did this by accident recently and broke systemd, but you might get away with it since you have a leading # (so it could be a comment)

either way, please remove this if you don't know why it's here, I don't think it's necessary.